src/pages/dashboard/component.js

```diff
@@ -1,23 +1,20 @@
 import React from 'react';
+import { SERVER } from 'constants.js';
+
+// Components
 import Heading from 'modules/Heading';
 import Banner from 'modules/Banner';
 import RecentProject from 'modules/RecentProject';
+
+// Image
 import background from 'img/city_banner.jpg';
 
+// Custom Hook
+import useFetch from 'customHooks/useFetch';
+
 
 const Dashboard = () => {
-  const projects = [
-    {
-      title: 'Project One',
-      description: 'Brief description here...',
-      id: 1,
-    },
-    {
-      title: 'Project Two: Project Manager',
-      description: 'Building a project management tool for freelancers to keep track of projects and clients.',
-      id: 2,
-    },
-  ];
+  const { data: projects, isPending, error } = useFetch(`${SERVER}/projects`);
```

Comment:
is this gonna fetch the data every time Dashboard re-renders?
you may want to limit it with a `useEffect()` hook

Reply:
I have a custom hook built called useFetch, where I am using the `useEffect()` to make the call.

Reply:
https://betterprogramming.pub/understanding-the-useeffect-dependency-array-2913da504c44